From 363695b326ce804836d89cf877289da95fb2281f Mon Sep 17 00:00:00 2001
From: Sidraya Jayagond <sidraya.bj@ti.com>
Date: Mon, 8 Mar 2021 19:35:48 +0530
Subject: [PATCH 2/2] Decoder driver hanging on Gstreamer and standalone test
 app

Commit hash "70f0e5762296848cde817752a648df3579ffbfb6" changes revoked to
avoid gstreamer launch command and standalone test app
hanging for indefinite time

Signed-off-by: Sidraya Jayagond <sidraya.bj@ti.com>
---
 driver/decoder/vxd_core.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/driver/decoder/vxd_core.c b/driver/decoder/vxd_core.c
index 791d470..1f984de 100644
--- a/driver/decoder/vxd_core.c
+++ b/driver/decoder/vxd_core.c
@@ -1477,9 +1477,7 @@ int32 vxd_send_msg(struct vxd_dec_ctx *ctx, struct vxd_fw_msg *msg)
 		"%s: added item %p to pend, ptd: 0x%x, str: %u flags: 0x%x\n",
 		__func__, item, stream->ptd, stream->id, item->msg.out_flags);
 
-	osa_mutex_lock(vxd->mutex);
 	vxd_schedule_locked(vxd);
-	osa_mutex_unlock(vxd->mutex);
 
 out_unlock:
 	osa_mutex_unlock(ctx->mutex);
-- 
2.17.1

